lib/src/cli.rs

```diff
@@ -991,7 +992,9 @@ impl Opt {
 
 /// Internal (non-generic/monomorphized) primary CLI entrypoint
 async fn run_from_opt(opt: Opt) -> Result<()> {
-    let root = &Dir::open_ambient_dir("/", cap_std::ambient_authority())?;
+    // We want to open *only* the root filesystem of the container image
+    // ie: without /sys, /proc, /etc/resolve.conf bind mounts, etc.
```

Comment:
This is a big change. I think in most cases it's right...OK actually this patch is just changing one of several copies of this we have and it's *definitely* safe and correct for the linter. But...